src/components/layout/Navbar.tsx

```diff
@@ -46,7 +48,7 @@ const Navbar = () => {
           <PopoverContainer label='Links' panelClasses='-left-6'>
             <NavbarDropdownContainer links={TOP_NAVIGATION.links} />
           </PopoverContainer>
-          <LoginWithGoogleButton />
+          {!path.startsWith('/register') && <LoginWithGoogleButton />}
```

Comment:
Don't hardcode '/register'. Check src/constant/routes.ts file and add Register route.